src/V2/Records/ListRelatedQuery.php

```diff
@@ -1,18 +1,13 @@
 <?php
 
-namespace Zoho\Crm\V2\RelatedRecords;
+namespace Zoho\Crm\V2\Records;
 
 use Zoho\Crm\Contracts\PaginatedQueryInterface;
 use Zoho\Crm\Contracts\ResponseTransformerInterface;
 use Zoho\Crm\Support\Helper;
 use Zoho\Crm\V2\Traits\HasPagination;
 
-/**
```

Comment:
You should not have removed the doc here.

Reply:
mmm... but that class doesn't exist right now...

Reply:
This file no, but the renamed one yes (src/V2/Records/ListRelatedQuery.php).